.github/workflows/build-web.yml

```diff
@@ -0,0 +1,65 @@
+name: 🕸️ Build (Web)
+
+on:
+  push:
+    branches:
+      - '**'
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - '**.md'
+      - '**.txt'
+      - '!**/CMakeLists.txt'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - '**.md'
+      - '**.txt'
+      - '!**/CMakeLists.txt'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    name: "\
+      ${{ matrix.platform }}-\
+      ${{ matrix.double == true && 'double-' || '' }}\
+      ${{ matrix.editor == true && 'editor-' || '' }}\
+      ${{ matrix.config }}"
+
+    strategy:
+      fail-fast: true
+      matrix:
+        platform: [web-wasm32]
+        double: [false] #, true]
+        editor: [false] #, true]
+        config: [debug] #, development, distribution]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup emsdk
+        uses: mymindstorm/setup-emsdk@v14
+        with:
+          version: 3.1.39
+          actions-cache-folder: 'emsdk-cache'
```

Comment:
Looking at [Godot's usage](https://github.com/godotengine/godot/blob/0a83e7c5dac550afb26e26684cbb6e4d6c139f5e/.github/workflows/web_builds.yml#L44-L49) of this GitHub Action I'm seeing them provide a custom `cache-key` as opposed to using [the default](https://github.com/mymindstorm/setup-emsdk/blob/6ab9eb1bda2574c4ddb79809fc9247783eaf9021/src/main.ts#L30) one, which to me looks like it essentially prevents caching inbetween different pull requests, since it includes the `github.ref` in it. I wonder why that is.

Reply:
Huh, interesting. Looks like they were having problems with cache contention with multiple concurrent builds. https://github.com/mymindstorm/setup-emsdk/issues/20#issuecomment-1898170886 

I do think maybe the branch name would have been sufficient, but I don't know too much about what and when they build.

Reply:
Right, well, that won't be a concern for this repository I guess, so might as well use the default key.